app/models/preference.rb

```diff
@@ -0,0 +1,7 @@
+class Preference < ApplicationRecord
```

Comment:
also forgot to mention:
we should add a validation for uniqueness so that the same user can't have a preference twice for the same genre